src/hotspot/share/classfile/compactHashtable.cpp

```diff
@@ -226,7 +226,7 @@ HashtableTextDump::HashtableTextDump(const char* filename) : _fd(-1) {
   if (_fd < 0) {
     quit("Unable to open hashtable dump file", filename);
   }
-  _base = os::map_memory(_fd, filename, 0, nullptr, _size, true, false);
+  _base = os::map_memory(_fd, filename, 0, nullptr, _size, mtSymbol, true, false);
```

Comment:
This seems to be used to read Symbols *OR* String. This probably needs to be something else. I suggest to revert to mtNone and figure out the appropriate tag later.
```suggestion
  _base = os::map_memory(_fd, filename, 0, nullptr, _size, mtNone, true, false);
```